scripts/check-expo-dependencies.js

```diff
@@ -0,0 +1,43 @@
+module.exports = async ({ context, github, fetch, dependencyNames }) => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const expoVersion = require('../package.json').dependencies.expo
+  const expoVersionSanitized = expoVersion.replace(/[!^~]/g, '')
```

Comment:
can we make the regxp slightly better?

1. why "!"? is that a valid version specifier?
2. why the "g" modifier? we know where the version specifier can occur and how many there can be. only one at the beginning of the version, so the regexp can be more specific